core/adapters/copy_test.go

```diff
@@ -80,16 +80,14 @@ func TestCopy_Perform(t *testing.T) {
 	}
 
 	for _, tt := range tests {
```

Comment:
Can we rename this to just `test` since we aren't assigning?

Reply:
I'm okay with it, but let's push things like that to a different PR, especially since this line wasn't changed here.